### ISSUE TITLE
CI test fails

### ISSUE THREAD

**Issue by ice1000** [agda/agda#5164]
It fails with

```
Files Issue720.out and Issue720.tmp differ
=== Old output ===
(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue720 (Issue720.agda). " t)
(agda2-highlight-add-annotations 'nil '(1 7 (keyword) t) '(17 22 (keyword) t))
(agda2-highlight-add-annotations 'nil '(1 7 (keyword) t) '(8 16 (module) nil nil ("Issue720.agda" . 1)) '(17 22 (keyword) t))
(agda2-highlight-add-annotations 'nil '(1 7 (keyword) t) '(8 16 (module) nil nil ("Issue720.agda" . 1)) '(17 22 (keyword) t))
(agda2-status-action "Checked")
(agda2-info-action "*All Done*" "" nil)
((last . 1) . (agda2-goals-action '()))
(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue720 (Issue720.agda). " t)
(agda2-highlight-add-annotations 'nil '(1 7 (keyword) t) '(17 22 (keyword) t))
(agda2-highlight-add-annotations 'nil '(1 7 (keyword) t) '(8 16 (module) nil nil ("Issue720.agda" . 1)) '(17 22 (keyword) t))
(agda2-highlight-add-annotations 'nil '(1 7 (keyword) t) '(8 16 (module) nil nil ("Issue720.agda" . 1)) '(17 22 (keyword) t))
(agda2-status-action "Checked")
(agda2-info-action "*All Done*" "" nil)
((last . 1) . (agda2-goals-action '()))
=== New output ===

Issue720.hs:«line»:«col»: error:
Could not load module ‘Agda.Version’
It is a member of the package ‘Agda-2.6.2-AltzoNQZhwF6I1cCJZseYT’
which is unusable due to missing dependencies:
aeson-1.5.5.1-8rCWEVqf1aUALYpAiicOsJ async-2.2.2-2HtfAkBhPPT8kseorSe4Sr bifunctors-5.5.10-1Xyw3zBBKdPGoolSEEYrSo blaze-html-0.9.1.2-94aIHAIThRl2ZOGzZzQu8I boxes-0.1.5-E9lnfyWcT9EKHWfZME6qRU case-insensitive-1.2.1.0-4OaE3RtEcce2PYQtpUYMQJ data-hash-0.2.0.1-JSaSL3K803yKsJpG4k0xw2 edit-distance-0.2.2.1-Fb0jZ2BiygGwMDEZ8jmef equivalence-0.3.5-4X2fsaWBFLlHM90qVxz19E gitrev-1.3.1-Het0FatRKM8GuBdILU3orB hashable-1.3.0.0-K3FFBNAmsvaIvBv4Qg15rQ hashtables-1.2.4.1-KLXwQaLBnUtDZtDeFP1sC haskeline-0.8.1.1-BNnTwHIFWOvIADoN63hFQ6 monad-control-1.0.2.3-FNg31cuSvxg1INhTbplQrp murmur-hash-0.1.0.9-GMMq0T3o2RfCFsSNchOE77 regex-tdfa-1.3.1.0-CcybAIKPz04GsiXSrn3MCn split-0.2.3.4-88UKBE2UY7dFTqHQfZTaFK strict-0.4.0.1-AgdP2vLyAmWC48ds2pDhhy text-icu-0.7.0.1-DIipBkytmEBLlGes9uBsFm unordered-containers-0.2.13.0-3awuPgUx2yvAACRZkw6am3 uri-encode-1.5.0.7-K5oSS0diKuBLtZ7KckO26g zlib-0.6.2.2-DNTDdnyQHg0DKqCy8HMznM
Use -v (or `:set -v` in ghci) to see a list of the files searched for.
|
7 | import Agda.Version
| ^^^^^^^^^^^^^^^^^^^
=== Diff ===
```

Any ideas why?

**Comment by UlfNorell** [comment]
See my comment on https://github.com/agda/agda/issues/5061#issuecomment-771645171.

**Comment by ice1000** [comment]
Oh, I see.

**Comment by L-TChen** [comment]
I am not so sure why stack may have a different build plan even for the same `stack.yaml`, so the cached build plan is different from the build plan supposed to use. I will use the [lock file](https://docs.haskellstack.org/en/stable/lock_files/) instead as the cache key. It seems fine on my fork https://github.com/L-TChen/agda/runs/1813818417?check_suite_focus=true. I will push this tomorrow.

**Comment by L-TChen** [comment]
I hope this issued is fixed by fec6097. Ping me if it happens again.
